app/src/pages/TopicsOverview.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import { appLanguageIdAsRef } from "@/globalConfig";
+import { db, DocType, TagType, type ContentDto } from "luminary-shared";
+import { ref, watchEffect } from "vue";
+
+// Fetch topics based on language
+const topics = db.tagsWhereTagTypeAsRef(TagType.Topic, { languageId: appLanguageIdAsRef.value });
+
+// Get content of topics
+const topicContent = ref<ContentDto[]>([]);
+
+watchEffect(async () => {
+    const topicId = topics.value.map((t) => t._id);
```

Comment:
Nitpick: Since it is a list of id's, this variable name is clearer when it's plural: `topicIds`.

```suggestion
    const topicIds = topics.value.map((t) => t._id);
```